projects/dotcms/src/lib/models/DotCMSContentType.model.ts

```diff
@@ -13,6 +15,7 @@ export interface DotCMSContentTypeField {
     dataType?: string;
     hint?: string;
     fieldTypeLabel?: string;
+    values?: string;
     variable?: string;
 }
```

Comment:
can we use here the interface define in angular?

Reply:
@fmontes what do you think?

Reply:
Even tho I think we should try to unify types somehow, in this case this is not a good approach because what we have in angular and this lib are completely different projects.